castervoice/apps/browser/chrome.py

```diff
@@ -0,0 +1,81 @@
+#
+# This file is a command-module for Dragonfly.
+# (c) Copyright 2008 by Christo Butcher
+# Licensed under the LGPL, see <http://www.gnu.org/licenses/>
+#
+"""
+Command-module for Chrome
+
+"""
+# ---------------------------------------------------------------------------
+
+from dragonfly import (Grammar, Repeat,
+                       Mouse, Pause)
+
+import browser
+from castervoice.apps.browser.browser_shared_commands import BrowserSharedCommands
+from castervoice.lib import control
+from castervoice.lib import settings
+from castervoice.lib.actions import Key, Text
+from castervoice.lib.context import AppContext
+from castervoice.lib.dfplus.merge import gfilter
+from castervoice.lib.dfplus.state.short import R
+from castervoice.lib.temporary import Store, Retrieve
+
+
+class ChromeRule(BrowserSharedCommands):
+    pronunciation = "google chrome"
+
+    _mapping = {
+        browser.OPEN_NEW_WINDOW:
+            R(Key("c-n")),
```

Comment:
This and many others is in `browser_shared_commands`. Why is it redefined here?

Reply:
what do you mean many others? This is the only duplicate right?